[FUCK] Forgotten Ship ruin fix 2: Policy config edition.

EgorDinamit opened the pull request:
## About The Pull Request

**First of all this PR adds config for ghost-role policy(#50185)**
**Also, it's going to fix and balance the forgotten ship ruin even more.**
**Cybersun Hardsuit armour values changed to:** _(**"melee" = 30**, **"bullet" = 40**, "laser" = 70, **"energy" = 70**, **"bomb" = 30**, "bio" = 100, **"rad" = 60**, **"fire" = 60**, "acid" = 60)_

## Why It's Good For The Game

In the current state, ruin may be disappointing and unbalanced.
Even though I'm not sure I can balance it to the ideal state, this fix also changes ruin's chance to spawn so we won't have it every round 24/7.
Furthermore, this PR adds Cybersun Hardsuit to traitors(14TC) and nuke ops(8TC) uplinks, so you don't have to ask admins for TC trade anymore.

## Changelog
:cl: EgorDinamit
add: Cybersun hardsuit added to traitors(14TC) and nuke ops(8TC) uplinks.
balance: Cybersun hardsuit armour values increased.
tweak: Adds and replaces a few items on the ship, such as firing pins, materials and encryption keys.
balance: Syndicate ship ruin turrets armour and rate of fire increased.
fix: Empty sleeper, that emits GPS signal is now indestructible.
config: Adds config to edit "important_info" for both, crewmembers and captain of forgotten ship ruin.
/:cl:

EgorDinamit committed:
Balance+Bug fix.

EgorDinamit committed:
Update forgottenship.dmm

EgorDinamit committed:
Update forgottenship.dmm

EgorDinamit committed:
Removes rogue table

EgorDinamit committed:
Give uranium to plebs.

EgorDinamit committed:
Beakers+Few chameleon stuff

EgorDinamit committed:
Fluff text rebalance. Allow people to leave the ship as long as it is protected.

EgorDinamit committed:
Grammar fix, I guess

EgorDinamit committed:
Le grammar

EgorDinamit committed:
Replaced interrogation room with medbay. Added tactical medkit(1) to med-vendor and 5 basic medkits to the same vendor. Additional ores, loot.

EgorDinamit committed:
Surgery disk update

EgorDinamit committed:
O2 canisters to maints area

EgorDinamit committed:
GPS signal

EgorDinamit committed:
Update code/game/objects/structures/ghost_role_spawners.dm

Co-Authored-By: Rohesie <rohesie@gmail.com>

EgorDinamit committed:
Main update

Changes ruin weight from 1 to 0.5.
--------------
Changes turrets armor in ship.
Shot delay of turrets changed from 15 to 8. (From 1.5 seconds to 0.8)
Replaces normal firing pins to syndicate ones in captain's locker.
Amount of uranium near generator changed from 50 to 20 so you actually have to build solars.
Replaced freeform AI core to cybersun variant.
--------------
Sleeper with gps signal in now indesctructible.
Cybersun hardsuit armor values increased.
Cybersun hardsuit added to traitor(14TC) and nuke ops uplink(For 8TC).

EgorDinamit committed:
Merge branch 'master' of git://github.com/tgstation/tgstation

EgorDinamit committed:
Add files via upload

EgorDinamit committed:
Add files via upload

EgorDinamit committed:
Add files via upload

EgorDinamit committed:
Add files via upload

EgorDinamit committed:
Merge branch 'master' into tgstation-master

TiviPlus commented on code/modules/ruins/spaceruin_code/forgottenship.dm:
```diff
@@ -33,18 +33,45 @@ GLOBAL_VAR_INIT(fscpassword, generate_password())
+	name = "box of syndicate firing pins"
+	desc = "A box full of special syndicate firing pins which allow only syndicate operatives to use weapons with those firing pins."
+	icon_state = "secbox"
+	illustration = "firingpin"
```
```suggestion
```

EgorDinamit replied:
Oh, the parent object has those already, right?

TiviPlus replied:
It does

Fikou commented:
the cybersun hardsuit now has 20 more laser armor than what's supposed to be the best anti laser armor in the game

EgorDinamit commented:
> the cybersun hardsuit now has 20 more laser armor than what's supposed to be the best anti laser armor in the game

60% armour + 50% chance to reflect lasers is still better, especially if you take in mind that disabler beams stamina damage isn't reduced by armour.

Let's just say, someone makes 4 shots with a laser gun(20 damage):
Cybersun: _Reduced from 80 to 16._
Reflector vest: [2 shots reflected, took only 2 shots instead of 4] _Reduced from 80(40) to 16._
So technically, they are somewhat the same, except cybersun one is a hardsuit.

EgorDinamit committed:
Update code/modules/ruins/spaceruin_code/forgottenship.dm

Co-Authored-By: TiviPlus <57223640+TiviPlus@users.noreply.github.com>

EgorDinamit commented:
Anyway, @Fikou , that's why we have an open conversation here - to share ideas.

spookydonut commented:
You need to fight against #50171, only one can live.

Timonkeyn commented:
i dont support this as long as there is no policy regarding the syndies

EgorDinamit commented:
> i dont support this as long as there is no policy regarding the syndies

I pinged some headmins about this on discord, soon enough there will be one.
Basically it means that syndies can do anything except for invading station/lavaland.

EgorDinamit commented:
Well then, I guess I'll change protection values to this instead:
**("melee" = 30, "bullet" = 40, "laser" = 70, "energy" = 50, "bomb" = 30, "bio" = 100, "rad" = 60, "fire" = 60, "acid" = 60)**

EgorDinamit committed:
Lowered Cybersun hardsuit armour values.

EgorDinamit committed:
Upgrading energy armour value.

EgorDinamit committed:
Now captain isn't male-only.

EgorDinamit committed:
Turrets

shot_delay changed from 0.8 seconds to 1.0 second.

EgorDinamit committed:
Adds 3 encryption keys for cyborgs and AIs to use

EgorDinamit committed:
Syndi-captain gets loud-voice headset.

EgorDinamit commented:
AppVeyor is kil, someone restart it later.

IndieanaJones commented:
I still this runs into the same problems as most ghost roles do, in that there's an entire lack of replayability on all fronts.  You get to sit in a ship that can't move all round, and very, very, rarely the outside world might come try to kill you.  Once you do this ruin once or twice, you have no reason to do it again.

EgorDinamit commented:
@IndieanaJones well, that's a ghost-roles, they aren't supposed to be full-time jobs, even though playing HoP or any other role for 100 hours might get boring at some point.
How fun it will be depends on the current round. 
**You got nuke ops?**: Why not build a teleporter and throw them some cool stuff(not people/borgs) to help them.
**You got smart traitors?**_(They bought encryption key)_: Help them navigate to the ship or if they have access to teleporter - print a beacon for a moment for them. It's even better if you got some syndie-ships and someone built gigabeacon on the station.
**Green shift?**: Green shift often means total boredom/autistic projects. That probably will double the chances for someone saying "Hey, let's raid this shit in space" or whatever.

At the same time, Charlie station is just shit. You might build a singulo engine, but I highly doubt that someone can do this without destroying everything in a 1-kilometre radius. Of course, you still can go to the station and, I dunno, be a wandering merchant? Ask HoP for a normal job here? Probably you'll just end up as "Someone who has omnilathe" for science or shady persons.

EgorDinamit committed:
Policy config

EgorDinamit committed:
Define space syndies

EgorDinamit committed:
Added Robotics access for syndies

It's really useful for when they create cyborgs.

EgorDinamit committed:
Adds special version of medivend instead of normal one

EgorDinamit committed:
Replaces basic syndi-vend with special one

EgorDinamit committed:
Update forgottenship.dm

EgorDinamit committed:
Cybersun hardsuit doesn't slow you down as said in uplink desc.

EgorDinamit committed:
NPC + Area sounds.

Elite Assault Officer projectile sound changed from pulse.ogg to laser.ogg.
Added ambient sounds for all areas.

EgorDinamit committed:
Oranges doesn't trust web-edit

EgorDinamit committed:
I guess we're not ready for curse.

EgorDinamit committed:
non-web edit

EgorDinamit committed:
Revert "non-web edit"

This reverts commit a54d257079315e46eec795a5d86623b7deb464d0, reversing
changes made to 664baf495fd3a03d00ce9267f1356b88233af226.

EgorDinamit committed:
Revert "I guess we're not ready for curse."

This reverts commit 664baf495fd3a03d00ce9267f1356b88233af226.

EgorDinamit committed:
Revert "Oranges doesn't trust web-edit"

This reverts commit 789a4531c80b1c04c0a76023d5e623f9e475508a.

EgorDinamit commented:
Oh god oh fuck

EgorDinamit committed:
Revert "Revert "Oranges doesn't trust web-edit""

This reverts commit 722d79bfedc8b16cd19912c35fbd5463eafafbfc.

EgorDinamit committed:
Revert "Revert "I guess we're not ready for curse.""

This reverts commit bc5e841cdd11eda931d388f284f7414a1349fd0b.

EgorDinamit committed:
Revert "Revert "non-web edit""

This reverts commit 98459f3f51fdceace6b6bc47b11dc745cacacfee.

EgorDinamit committed:
Merge branch 'master' of https://github.com/tgstation/tgstation

EgorDinamit committed:
Merge branch 'master' into tgstation-master

EgorDinamit commented:
Shit's fucked

EgorDinamit commented:
I'll open a new one, *sigh*
